pennylane_qiskit/qiskit_device.py

```diff
@@ -152,40 +157,29 @@ def process_kwargs(self, kwargs):
         Args:
             kwargs (dict): keyword arguments to be set for the device
         """
-        aer_provider = str(self.provider) == "AerProvider"
```

Comment:
How come these were removed? It seems that there were no relevant changes to the providers